Core/RadioHandler.cpp

```diff
@@ -74,27 +68,40 @@ RadioHandlerClass::RadioHandlerClass() :
 	inputbuffer.setBlockSize(transferSamples);
 
 	stateFineTune = new shift_limited_unroll_C_sse_data_t();
+	adcrate = adcnominalfreq;
+	fprintf(stderr,"RadioHandlerClass::RadioHandlerClass\n");
```

Comment:
use DbgPrintf instead. also I am not sure if this log is still useful.

Reply:
I had segfaults, freezes and completely wrong output. Many debug messages were added to fix segfaults (double free, memory corruptions). I was really surprised by the fact, that this library works under windows. I even thought that the windows library is built from some other private repository...

Reply:
No. We are using the public repo. there is no private one.